src/pages/home/report/ReportActionCompose.js

```diff
@@ -207,9 +211,12 @@ class ReportActionCompose extends React.Component {
                         )}
                     </AttachmentModal>
                     <TouchableOpacity
-                        style={[styles.chatItemSubmitButton, styles.buttonSuccess]}
+                        style={[styles.chatItemSubmitButton,
+                            (this.state.isCommentEmpty || this.props.comment)
```

Comment:
We should be checking for `!this.state.isCommentEmpty` here. Also there is probably no need to check `this.props.comment` as well if we initialize the state value from `props`.

Reply:
Removed this.props.comment now only check is now with the state.